servers/quarkus-server/src/test/java/org/projectnessie/server/profiles/QuarkusNativeProfileInmemory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server.profiles;
+
+import com.google.common.collect.ImmutableMap;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class QuarkusNativeProfileInmemory implements QuarkusTestProfile {
+
+  @Override
+  public Map<String, String> getConfigOverrides() {
+    return ImmutableMap.of("nessie.version.store.type", "INMEMORY");
+  }
+
+  @Override
+  public String getConfigProfile() {
+    return "prod";
```

Comment:
How does this work with unit test that generally use the `test` quarkus profile, if I'm not mistaken?

Reply:
native-tests require the `prod`, not the `test` profile, sadly

Reply:
yeah, I meant using this in `TestBasicOperations`, which is fixed now :+1: